alvr/dashboard/src/dashboard/components/setup_wizard.rs

```diff
@@ -1,7 +1,7 @@
 use crate::dashboard::basic_components;
 use alvr_common::{error, warn};
 use alvr_packets::{FirewallRulesAction, PathValuePair, ServerRequest};
-#[cfg(not(target_os = "linux"))]
+#[cfg(target_os = "win64")]
```

Comment:
You should use `#[cfg(windows)]

Reply:
Actually, for the import, I'd prefer to qualify the objects/functions inline when they are cfg-gated, so the imports at the top of the file look cleaner

Reply:
what about in-function per-platform imports?

Reply:
in that case I'd add the `use` statement at the top of the function, or at the top of the cfg'd block.